src/stores/rebus/query/validator-voting-record/types.ts

```diff
@@ -0,0 +1,31 @@
+type PublicKey = {
```

Comment:
You should delete copied over types that youre not using here

Reply:
this was all removed now